src/main/resources/db/migration/refdata/V1_19__update_disabilities.sql

```diff
@@ -0,0 +1,20 @@
+-- add individual disability questions
```

Comment:
Note that there'll be a filename conflict here - there's already a V1_19 waiting to be merged in https://github.com/ministryofjustice/hmpps-assessments-api/pull/492

Reply:
Yup! I'll rebase and merge once `ARN-1040` is merged 👍